src/utils/filter-utils.js

```diff
@@ -698,13 +698,23 @@ export function getNumericFieldDomain(data, valueAccessor) {
   return {domain, step, histogram, enlargedHistogram};
 }
 
-function getNumericStepSize(diff) {
+export function getNumericStepSize(diff) {
+  diff = Math.abs(diff);
+
   if (diff > 100) {
     return 1;
-  } else if (diff < 20 && diff > 3) {
+  } else if (diff > 3) {
     return 0.01;
-  } else if (diff <= 3) {
+  } else if (diff > 1) {
     return 0.001;
+  } else if (diff <= 1) {
+    // Try to get at least 100 steps - and keep the step size below that of
+    // the (diff > 1) case.
+    const x = diff / 100;
```

Comment:
can you make this min step to be 1000?

Reply:
Sure, updated. I also updated tests based on the values I saw